firebase_admin/ml.py

```diff
@@ -440,8 +440,18 @@ def _parse_gcs_tflite_uri(uri):
     def upload(bucket_name, model_file_name, app):
         """Upload a model file to the specified Storage bucket."""
         _CloudStorageClient._assert_gcs_enabled()
+
+        # Calculate the destination file_name (remove path if present)
+        file_name = model_file_name
```

Comment:
This whole thing can be condensed to:

```
file_name = os.path.basename(model_file_name)
```

That should work in Windows environments too (that's probably not a requirement, but just in case)

Reply:
Nice! done.